app/models/jobsModel.js

```diff
@@ -1,22 +1,59 @@
 const db = require('../database/dbConfig.js')
+const crypto = require('crypto')
 
 /**
  * Inserts a job into the database.
  * scheduled_time is an optional field.
  * 
  * @param {"class_id" : integer, 
- *         "scheduled_time" : Date, 
- *         "job_hash" : string} job 
+ *         "scheduled_time" : Date} job 
  */
 async function insertJob(job){
```

Comment:
Since this is the jobs model, we know the functions have something to do with jobs for sure. You can just name this function insert